tuplan_garage/planning/training/modeling/models/pdm_offset_model.py

```diff
@@ -173,7 +192,11 @@ def build_model(self):
         config_p.hidden_size = config_p.n_embd
         config_p.intermediate_size = config_p.n_inner
         config_p.num_attention_heads = config_p.n_head
+        config_p.n_action_steps = 80
```

Comment:
elaborate the reason for adding n_action_steps?

Reply:
Yes, it is not necessary